I've just tried this PR locally and I get the following error:

```
utils/regex.cc: In member function 'int modsecurity::Utils::Regex::search(const string&, modsecurity::Utils::SMatch*) const':
utils/regex.cc:294:24: error: 'rc' was not declared in this scope
     if (m_pcje != 0 || rc == PCRE2_ERROR_JIT_STACKLIMIT) {
                        ^~
utils/regex.cc:294:24: note: suggested alternative: 'rcmd'
     if (m_pcje != 0 || rc == PCRE2_ERROR_JIT_STACKLIMIT) {
                        ^~
                        rcmd
```

Adding an "int rc;" fixes things

--- a/src/operators/verify_cc.h
+++ b/src/operators/verify_cc.h
@@ -39,7 +39,12 @@ class VerifyCC : public Operator {
     explicit VerifyCC(std::unique_ptr<RunTimeString> param)
         : Operator("VerifyCC", std::move(param)),
 #if WITH_PCRE2
-        m_pc(NULL) { }
+        m_pc(NULL) 
+		{
+#if WITH_PCRE2
+			m_pcje = PCRE2_ERROR_JIT_BADOPTION;
+#endif
+		}
 #else
         m_pc(NULL),
         m_pce(NULL) { }

--- a/src/utils/regex.cc
+++ b/src/utils/regex.cc
@@ -291,7 +291,7 @@ int Regex::search(const std::string& s, SMatch *match) const {
             0, 0, match_data, NULL) > 0;
     } 
     
-    if (m_pcje != 0 || rc == PCRE2_ERROR_JIT_STACKLIMIT) {
+    if (m_pcje != 0 || ret == PCRE2_ERROR_JIT_STACKLIMIT) {
         ret = pcre2_match(m_pc, pcre2_s, s.length(),
             0, PCRE2_NO_JIT, match_data, NULL) > 0;
     }

